include/circt/Dialect/SV/SVStatements.td

```diff
@@ -298,6 +298,11 @@ def CaseZStmt: I32EnumAttrCase<"CaseZStmt", 2, "casez">;
 def CaseStmtTypeAttr : I32EnumAttr<"CaseStmtType", "case type",
                                    [CaseStmt, CaseXStmt, CaseZStmt]>;
 
+def UniquePriorityUnique: I32EnumAttrCase<"UniquePriorityUnique", 0, "unique">;
+def UniquePriorityUnique0: I32EnumAttrCase<"UniquePriorityUnique0", 1, "unique0">;
+def UniquePriorityPriority: I32EnumAttrCase<"UniquePriorityPriority", 2, "priority">;
+def UniquePriorityTypeAttr: I32EnumAttr<"UniquePriorityType", "unique priority type",
```

Comment:
Nit: is there a better name for this? The SV spec hints at "case qualifier" or "validation qualifier".

Reply:
Thanks. I'd use "validation qualifier" since these constructs are also applicable to if statements.